### PR TITLE
Fix/object detail recordcell tooltip

### PR DESCRIPTION
**Pull request by Sandheep-OSC**
Fixes for https://github.com/twentyhq/twenty/issues/6596
image

### CODE REVIEW

File: packages/twenty-front/src/modules/ui/field/display/components/EllipsisTextWithTooltip.tsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
+import { StyledEllipsisDisplay } from '@/ui/field/display/components/EllipsisDisplay';
```
Comment (martmull) [inline packages/twenty-front/src/modules/ui/field/display/components/EllipsisTextWithTooltip.tsx:1]: looks like this component already exists here `packages/twenty-ui/src/display/tooltip/OverflowingTextWithTooltip.tsx`

File: packages/twenty-front/src/modules/object-record/record-inline-cell/components/RecordInlineCellContainer.tsx [hunk 2]
```diff
@@ -120,7 +120,7 @@ export const RecordInlineCellContainer = () => {
           )}
           {showLabel && label && (
             <StyledLabelContainer width={labelWidth}>
-              <EllipsisDisplay maxWidth={labelWidth}>{label}</EllipsisDisplay>
+              <EllipsisTextWithTooltip text={label} labelId={labelId} />
```
Comment (martmull) [inline packages/twenty-front/src/modules/object-record/record-inline-cell/components/RecordInlineCellContainer.tsx:123]: think you can use existing component `<OverflowingTextWithTooltip ... />`

**Review by martmull — CHANGES_REQUESTED** [review]

File: packages/twenty-front/src/modules/object-record/record-inline-cell/components/RecordInlineCellContainer.tsx (hunk 2 above)
Comment (lucasbordeau) [reply packages/twenty-front/src/modules/object-record/record-inline-cell/components/RecordInlineCellContainer.tsx:123]: Yes I forgot we already have one !

Comment (lucasbordeau) [reply packages/twenty-front/src/modules/object-record/record-inline-cell/components/RecordInlineCellContainer.tsx:123]: Let's use `OverflowingTextWithTooltip`then.

**Review by martmull — APPROVED** [review]

**Pull request merged**